src/systemd/sd-json.h

```diff
@@ -341,6 +342,10 @@ int sd_json_variant_unhex(sd_json_variant *v, void **ret, size_t *ret_size);
 const char* sd_json_variant_type_to_string(sd_json_variant_type_t t);
 sd_json_variant_type_t sd_json_variant_type_from_string(const char *s);
 
+static inline bool sd_json_enabled(sd_json_format_flags_t flags) {
+        return !(flags & SD_JSON_FORMAT_OFF);
+}
```

Comment:
i wonder if we shoud uppercase this, i.e. make this "macro-like"?

SD_JSON_FORMAT_ENABLED()?

or even

SD_JSON_FORMAT_IS_ENABLED()?

Reply:
We don't use upper case in other places in our headers. And I don't think we should: those are _not_ macros. The convention to use UPPERCASE for macros is because macros tend to behave differently than function calls, in particular because of possible multiple expansion of arguments or even zero expansion. But inline function calls behave very much like normal function calls.